apps/prairielearn/src/pages/instructorQuestionSettings/instructorQuestionSettings.html.ts

```diff
@@ -187,7 +189,13 @@ export function InstructorQuestionSettings({
                                     value="${topic.name}"
                                     data-color="${topic.color}"
                                     data-name="${topic.name}"
-                                    data-description="${topic.description}"
+                                    ${topic.implicit
+                                      ? ''
+                                      : html`
+                                          data-description="${TopicDescription({
+                                            topic_description: topic,
+                                          })}"
+                                        `}
```

Comment:
Nit, Prettier is better able to handle things like this. This itself will need formatting by Prettier. Ditto for tag descriptions.

```suggestion
                                    data-description="${topic.implicit ? '' : TopicDescription({
                                      topic_description: topic,
                                    })}"
```

Reply:
Fixed in 3680547